exceptiongen/__init__.py,sha256=hEP2KHd_6a6XuJK9Nstr-8uEczxrfNTANwwQuNiX8Dc,457
exceptiongenerator-2.2.dist-info/DESCRIPTION.rst,sha256=OCTuuN6LcWulhHS3d5rfjdsQtW22n7HENFRh6jC6ego,10
exceptiongenerator-2.2.dist-info/METADATA,sha256=kR2atxhtVch5_YIBl9mEPrs1j1o6VEI_aVq-Uk_KmXE,281
exceptiongenerator-2.2.dist-info/RECORD,,
exceptiongenerator-2.2.dist-info/WHEEL,sha256=BTHHj6QhsHpWhRUrxioIvCAgjx81u-a8gkPTO6bi_A0,116
exceptiongenerator-2.2.dist-info/metadata.json,sha256=dJ0pQWU4X5lEfLd-EpDikq7hMw_odP9CgTunob9SeKw,457
exceptiongenerator-2.2.dist-info/top_level.txt,sha256=vjzNUzV67RifbiU_t2Fbj2Si8c0By8uyEsC1SNZABBU,13
exceptiongenerator-2.2.dist-info/zip-safe,sha256=frcCV1k9oG9oKj3dpUqdJg1PxRT2RSN_XKdLCPjaYaY,2
